HW1/P5/P5.txt

```diff
@@ -1 +1,4 @@
 # Your discussion here
+
+['Harvard_University','Barack_Obama','Kevin_Bacon']
+[‘Kevin_Bacon’,’Barack_Obama’,’Harvard_University’]
```

Comment:
I think @isadoranun is correct, that the reverse is not true.  I suspect there might be something wrong in your P5?